service/src/test/java/bio/terra/cbas/dao/TestMethodDao.java

```diff
@@ -134,4 +135,22 @@ void unsetLastRunSetId() {
     Method updatedMethod = methodDao.getMethod(retrievedMethod.methodId());
     assertNull(updatedMethod.lastRunSetId());
   }
+
+  @Test
+  void deleteMethod() {
+    List<Method> allMethods = methodDao.getMethods();
+    assertEquals(2, allMethods.size());
```

Comment:
It's probably better to assume that each test starts with an empty database and sets up its own state, instead of relying on either previous tests (because order isn't guaranteed) or some initializer (because that locks tests to using the same initial state).

Otherwise when I look at this test case my main thought is... wait, why should I expect there to be 2 methods already in there?

If you're worried about code copying, you could always just make the "initialize state" method called explicitly instead of implicitly